pkg/monitoring/client/prometheus/prometheus.go

```diff
@@ -82,3 +114,75 @@ type resultAdapter struct {
 func (r *resultAdapter) Increment(code, method, host string) {
 	r.m.WithLabelValues(code, method, host).Inc()
 }
+
+type rtWrapper struct {
+	origRoundTripper http.RoundTripper
+}
+
+func parseURLResourceOperation(request *http.Request) (resource string, operation string) {
+	//pathSplit := strings.Split(request.URL.Path, "/")
```

Comment:
It might be here because you were debugging...

Reply:
oops, thanks